Examples/Autoregressive/fast_autoreg_sample.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
```

Comment:
I would simplify this removing all the benchmarking instructions etc, just to keep it a simple example as the others in the folder

Reply:
since you already wrote it, you  could keep a version of the file with all benchmarking instruction sin netket/benchmarks.